src/index.html

```diff
@@ -6,9 +6,24 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@800&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <a href="index.html" class="a_img"><img src="images/logo.png" alt="The logo of Moyo"></a>
+      <nav class="menu_nav">
+        <ul class="head_list">
+          <li><a href="#" class="is-active list_link">Apple</a></li>
```

Comment:
please add spaces between the <li> blocks as it will make a code cleaner

Reply:
Could you please confirm if I understood right. Do I need to put `<a>` tag  into new line, or I need to add empty lines after all tags?

Reply:
you need to add empty lines after all tags
if some <li> is to long, it's better to move <li> content to the next line